packages/core/src/Flex/examples/FlexExample.tsx

```diff
@@ -1,28 +1,40 @@
 import React from 'react';
-import { Flex } from '@contentful/f36-components';
+import { Box, Flex } from '@contentful/f36-components';
 import tokens from '@contentful/f36-tokens';
 
 export default function FlexExample() {
-  const ColorBox = ({ backgroundColor, children }) => (
-    <Flex
-      justifyContent="center"
-      alignItems="center"
-      style={{
-        width: '60px',
-        height: '60px',
-        color: tokens.colorWhite,
-        backgroundColor,
-      }}
-    >
-      {children}
-    </Flex>
-  );
-
   return (
     <Flex justifyContent="space-between" alignItems="center">
-      <ColorBox backgroundColor={tokens.green500}>Green</ColorBox>
-      <ColorBox backgroundColor={tokens.red500}>Red</ColorBox>
-      <ColorBox backgroundColor={tokens.blue500}>Blue</ColorBox>
+      <Box
+        justifyContent="center"
+        alignItems="center"
```

Comment:
`Box` doesn’t have flex styles, same comment for all other places
```suggestion
```

Reply:
Exactly, I'd keep ColorBox component here, but we can use `<div>` and center text differentely

Reply:
sure, overlooked. sorry. 
But I think we don't need text inside boxes at all.